### ISSUE TITLE
Corrupt graphics using proton on wine

### ISSUE THREAD

**Issue by Blad3forc3** [xenia-project/xenia#2072]
### Validation

- [X] I've read the [FAQ](https://github.com/xenia-project/xenia/wiki/FAQ).
- [X] The Xenia build used is from the master branch (not MLBS/AlexVS/Canary/pull requests, etc.)
- [X] This issue isn't for tech support (help with Xenia).
- [X] If this issue occurs in a specific game, I've done analysis to locate the faulty subsystem of the emulator and a potential reason in it.
- [X] I've checked if this issue hasn't already been reported.
- [X] My device meets the minimum requirements: https://github.com/xenia-project/xenia/wiki/Quickstart#system-requirements
- [ ] (If building) I have read the building doc: https://github.com/xenia-project/xenia/blob/master/docs/building.md

### Describe what's going wrong

Running games in xenia produces this problem with the graphics using proton in wine

![Screenshot_20221004_190215](https://user-images.githubusercontent.com/39629883/193892972-1086a4f6-cf1b-49b1-b0eb-5a9beccc2049.png)

### Describe what should happen

Game runs without graphics problems

**Comment by Razzile** [comment]
See https://github.com/xenia-project/xenia/issues/2045#issuecomment-1265183784 for why this is. 

tl;dr it's Proton at fault not Xenia
